crates/compiler/src/obj_gen/mod.rs

```diff
@@ -1131,8 +1182,23 @@ impl ObjectGenerator {
         bb: &mut BlockBuilder,
         func_ctx: &mut FunctionContext,
     ) -> Result<()> {
-        #[allow(clippy::enum_glob_use)] // We use all variants here.
-        use LLVMType::*;
+        use LLVMType::{
```

Comment:
Not sure these are better than the globs. I'd consider going back to either the globs or writing everything out, but your call.